meta/SchemaBuilder.php

```diff
@@ -199,6 +202,26 @@ protected function updateColumns() {
         return true;
     }
 
+    private function migrateSingleToMulti($table, $colref) {
+        $sqlSelect = "SELECT pid, rev, col$colref AS value FROM data_$table WHERE latest = 1";
+        $res = $this->sqlite->query($sqlSelect);
+        $valueSet = $this->sqlite->res2arr($res);
+        $this->sqlite->res_close($res);
+        $valueString = array();
+        foreach ($valueSet as $values) {
+            if (blank($values['value']) || trim($values['value']) == '') {
+                continue;
+            }
+            $valueString[] = "($colref, '$values[pid]', $values[rev], 1, '$values[value]')";
+        }
+        if (empty($valueString)) {
+            return;
+        }
+        $valueString = join(',', $valueString);
+        $sqlInsert = "INSERT OR REPLACE INTO multi_$table (colref, pid, rev, row, value) VALUES $valueString";
```

Comment:
Yes, I incorrectly assumed that since I handle DB-values, not direct user input, that escaping would be unnecessary. But that turned out to be wrong.

This should be fixed with the latest commit.